src/Symfony/Bundle/FrameworkBundle/Tests/Command/CacheClearCommand/Fixture/.preload.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if (file_exists(__DIR__.'/test/var/cache/test/Symfony_Bundle_FrameworkBundle_Tests_Command_CacheClearCommand_Fixture_TestAppKernelTestDebugContainer.preload.php')) {
```

Comment:
why is this here? maybe a rebase would remove it?

Reply:
sorry. I interrupted a test suite and didn't noticed a file was added